### PR TITLE
ensure swagger docs route matches the configured new api route

### LINKED ISSUE #11483: Hardcoded `new-api` route in Swagger Documentation normalizers
**Sylius version affected**: 1.8-dev

**Description**  
in 1.8 the api route can be configured, but the swagger documentation normalizers
`\Sylius\Bundle\ApiBundle\Swagger\ShopAuthenticationTokenDocumentationNormalizer` and `\Sylius\Bundle\ApiBundle\Swagger\AdminAuthenticationTokenDocumentationNormalizer` hardcode `/new-api` as path, resulting in swagger sending requests to an invalid path when you change the api route in your own configuration

**Steps to reproduce**  
update `security.yaml` paramter `sylius.security.new_api_route` to `/some-api`
navigate to `/some-api/docs` and you will see the token routes still displayed as `/new-api` so swagger will call those resulting in a 404

**Possible Solution**  
Inject the parameter as argument into the services (but maybe better to have a bundle config that then defines the parameter for you that is then used)

### PR DESCRIPTION
**Pull request by esserj**
fixes #11483

| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #11483
| License         | MIT

when you change the new api route in the parameters, it will now correctly reflect in the documentation

### CODE REVIEW

File: src/Sylius/Bundle/ApiBundle/Swagger/AdminAuthenticationTokenDocumentationNormalizer.php [hunk 1]
```diff
@@ -20,10 +20,15 @@ final class AdminAuthenticationTokenDocumentationNormalizer implements Normalize
 {
     /** @var NormalizerInterface */
     private $decoratedNormalizer;
+    /**
+     * @var string
+     */
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ApiBundle/Swagger/AdminAuthenticationTokenDocumentationNormalizer.php:25]:
```suggestion

    /** @var string */
```

File: src/Sylius/Bundle/ApiBundle/Swagger/ShopAuthenticationTokenDocumentationNormalizer.php [hunk 2]
```diff
@@ -20,10 +20,15 @@ final class ShopAuthenticationTokenDocumentationNormalizer implements Normalizer
 {
     /** @var NormalizerInterface */
     private $decoratedNormalizer;
+    /**
+     * @var string
+     */
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ApiBundle/Swagger/ShopAuthenticationTokenDocumentationNormalizer.php:25]:
```suggestion

    /** @var string */
```

**Comment by lchrusciel** [comment]
Hello Jan!

Thank you very much for your contribution :)

**Comment by esserj** [comment]
@lchrusciel your most welcome, thanks for making an awesome e-commerce platform!

committed the oneline comment suggestions

**Comment by lchrusciel** [comment]
Hello Jan,

could you rebase to the newest master? The build should be fixed then

**Comment by esserj** [comment]
🤞

**Comment by lchrusciel** [comment]
Thank you, Jan! :tada:

**Pull request merged**